core/services/ocr2/plugins/ocr2keeper/evm21/registry.go

```diff
@@ -601,6 +577,7 @@ func (r *EvmRegistry) doCheck(ctx context.Context, mercuryEnabled bool, keys []o
 		}
 	}
 
+	r.lggr.Infof("doCheck upkeepResults=%v", upkeepResults)
```

Comment:
Leftover from debugging or do we want to print the results?
if we want to print the results let's print only identifiers or stuff that won't overload the logs